keras_core/ops/image.py

```diff
@@ -244,3 +245,177 @@ def affine_transform(
         fill_value=fill_value,
         data_format=data_format,
     )
+
+
+class ExtractPatches(Operation):
+    def __init__(
+        self,
+        size,
+        strides=None,
+        rates=1,
```

Comment:
Is this the same as the `dilation_rate` arg in conv? If so, it should be renamed `dilation_rate`.